Detectors/HMPID/workflow/src/DataDecoderSpec2.cxx

```diff
@@ -341,7 +345,8 @@ o2::framework::DataProcessorSpec getDecodingSpec2(bool askDISTSTF)
     inputs,
     outputs,
     AlgorithmSpec{adaptFromTask<DataDecoderTask2>()},
-    Options{{"result-file", VariantType::String, "/tmp/hmpRawDecodeResults", {"Base name of the decoding results files."}},
+    Options{{"get-results-statistics ", VariantType::Bool, false, {"Generate intermediat output results."}},
```

Comment:
Here is a typo, there is a trailing space in the option name which should be removed.